packages/rolldown/src/options/input-options.ts

```diff
@@ -98,6 +98,18 @@ const inputOptionsSchema = z.strictObject({
   inject: z.record(z.string().or(z.tuple([z.string(), z.string()]))).optional(),
 })
 
+export const inputCliOptionsSchema = inputOptionsSchema.extend({
+  input: z.array(z.string()).optional(),
+  external: z.array(z.string()).optional(),
+  onLog: z.string().optional(), // command
+  onwarn: z.string().optional(),
+  inject: z.record(z.string()).optional(),
+  // Doesn't support treeshake options as it is ambiguous
+  treeshake: z.undefined().optional(),
+  // Doesn't support plugin yet.
+  plugins: z.undefined().optional(),
+})
+
```

Comment:
In command lines, options don't support functions, so I override these function-accepted options and just allow accepted types.